chains/metadata.yaml

```diff
@@ -770,10 +770,10 @@ confluxespace:
     - http: https://evm.confluxrpc.com
 connextsepolia:
   blockExplorers:
-    - apiUrl: https://connext-sepolia.blockscout.com/api
+    - apiUrl: https://scan.testnet.everclear.org/api
```

Comment:
Also these unrelated changes shouldn't be showing up here. Try merging in the latest `main` to your branch